nmosnode/nodefacadeservice.py

```diff
@@ -109,27 +107,20 @@ def __init__(self, interactive=False):
             "self": "ver_slf"
         }
         self.mdns_updater = None
-        if HTTPS_MODE == "enabled" and ENABLE_P2P:
+        self.auth_registry = AuthRegistry()
+        self.aggregator = Aggregator(self.logger, self.mdns_updater, self.auth_registry)
```

Comment:
I think this line may need to go lower. The mdns_updater won't be initialised yet here.